src/ServiceControl.Config/UI/Shell/VersionCheckerHelper.cs

```diff
@@ -0,0 +1,90 @@
+﻿namespace ServiceControl.Config.UI.Shell
+{
+    using Newtonsoft.Json;
+    using ServiceControlInstaller.Engine.Configuration.ServiceControl;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Threading.Tasks;
+
+    public static class VersionCheckerHelper
+    {
+        public static async Task<Release> GetDownloadUrlForNextVersionToUpdate(string currentVersion)
```

Comment:
I would consider changing this name to something like `GetRecommendedRelease()`.